src/chunk.h

```diff
@@ -186,6 +186,7 @@ extern void ts_chunks_rename_schema_name(char *old_schema, char *new_schema);
 
 extern TSDLLEXPORT bool ts_chunk_set_unordered(Chunk *chunk);
 extern TSDLLEXPORT bool ts_chunk_set_frozen(Chunk *chunk);
+extern TSDLLEXPORT bool ts_chunk_undo_frozen(Chunk *chunk);
```

Comment:
Nit. The name is a bit weird, what about something like `ts_chunk_unset_frozen` or `ts_chunk_set_unfrozen`

Reply:
renamed.